src/reference-data/defaults/city-zones-message-texts.json

```diff
@@ -0,0 +1,20 @@
+{
+  "singleZone": {
+    "message": [
+      {"lang": "nob", "value": "Bestillingstransport tilgjengelig i {0}."},
+      {
+        "lang": "eng",
+        "value": "Ordered transport is available in {0}."
+      }
+    ]
+  },
+  "multipleZones": {
+    "message": [
+      {
+        "lang": "nob",
+        "value": "Bestillingstransport tilgjengelig i {0} og {1}."
+      },
+      {"lang": "eng", "value": "Ordered transport is available in {0} and {1}."}
+    ]
+  }
+}
```

Comment:
I feel these texts is more like every other "normal" text we have in the app, which is not specified in Firestore. Since we don't have "normal" texts in Firestore, it also means we don't have an existing generic solution for specifying texts outside of the app.

Here the reference data is used for this purpose, which might be ok for a quick fix, but I don't feel it is good for a long term solution. Also this implementation introduces the possibility to inject strings into the text, here with the placeholders `{0}` and `{1}` and supplementing logic in the method `getTextForLanguageWithFormat`:
```
for (let index = 0; index < values.length; index++) {
  text = text.replace(`{${index}}`, values[index]);
}
``` 

This also introduces a risk, since the template in firestore might come out of sync with the usage in the app. 

I'll also tag @tor

To summarise this comment into a question: Do we really need to have these texts in Firestore? Are they more important to be dynamic than other texts we have in the app?

I'll also add @tormoseng and @oyvindmikkelsenatb in case this have already been discussed with you.

Reply:
@oyvindmikkelsenatb Base on the talk we had where the text were approved, is this still needed, be able to customize this form outside? With this, we wanted to have it open for later fixes and changes without make a new release @gorandalum, but everything has changed a lot. 

Regarding the function, it might be that from Firebase there is no ${0} or ${1} but the app will not crash since there will not be text to replace, the values are always passed by the app.

Reply:
@gorandalum We had a meeting, and we agree that it could be removed now. So I will do it.